salt/states/pkg.py

```diff
@@ -1241,10 +1245,14 @@ def installed(
     failed_hold = None
     if targets or to_reinstall:
         reinstall = bool(to_reinstall)
+        force = False
+        if salt.utils.is_freebsd():
+            force = True    # Downgrades need to be forced.
```

Comment:
@terminalmage You're welcome.  This is my first time looking at the salt code, so I wasn't aware of the yumpkg downgrade stuff.  If you need any more changes from me in a follow-up pull request please let me know.